.github/workflows/pr-project-board.yml

```diff
@@ -20,3 +21,4 @@ jobs:
       - name: Move PR to Project Board for triage
         run: |
           gh pr edit ${{github.event.pull_request.number}} --add-project "NR1 Quickstarts Community"
+          COLUMN=$(gh api -H "$HEADER" projects/$PROJECT_ID/columns --jq ".[] | select(.name == \"$COLUMN_NAME\").id")
```

Comment:
i think we will want to use the whole code block from the other workflow:
```sh
        PR_NUMBER=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
        HEADER="Accept: application/vnd.github.inertia-preview+json"
        PR_ID=$(gh api -H "$HEADER" "repos/$REPOSITORY/pulls?state=open" --jq ".[] | select(.number == $PR_NUMBER).id")
        PROJECT=$(gh api -H "$HEADER" "repos/$REPOSITORY/projects" --jq ".[] | select(.id == $PROJECT_ID).id")
        COLUMN=$(gh api -H "$HEADER" "projects/$PROJECT/columns" --jq ".[] | select(.name == \"$COLUMN_NAME\").id")
        gh api -H "$HEADER" -X POST "projects/columns/$COLUMN/cards" -f content_type='PullRequest' -F content_id="$PR_ID"
```
and remove `gh pr edit . . .`